### PR TITLE
python3Packages.hiredis-py: init at 1.0.0

### PR DESCRIPTION
**Pull request by alex-ameen**
###### Motivation for this change

Require package for a project.

###### Things done

Added python package "hiredis-py" to repo.

- [ ] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [x] macOS
   - [x] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nix-review --run "nix-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Assured whether relevant documentation is up to date
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

File: pkgs/development/python-modules/hiredis-py/default.nix [hunk 1]
```diff
@@ -0,0 +1,24 @@
+{ stdenv
+, fetchurl
+, buildPythonPackage
+}:
+buildPythonPackage rec {
```
Comment (matthiasbeyer) [inline pkgs/development/python-modules/hiredis-py/default.nix]: Not sure whether you need `rec` here.

**Review by matthiasbeyer — CHANGES_REQUESTED** [review]
Also, please do not add yourself as a maintainer in the same commit, please split that into a seperate commit!

File: pkgs/development/python-modules/hiredis-py/default.nix (hunk 1 above)
Comment (BadDecisionsAlex) [reply pkgs/development/python-modules/hiredis-py/default.nix]: I had removed `rec` and split adding myself to maintainers into another commit.

**Comment by matthiasbeyer** [comment]
You are still adding yourself in the maintainers file in a commit that also alters a package... please split that commit.

**Comment by BadDecisionsAlex** [comment]
> You are still adding yourself in the maintainers file in a commit that also alters a package... please split that commit.

I separated them into two commits in this PR and my other open PRs. Are you requesting that I make a separate PR to add myself as a maintainer?

**Comment by matthiasbeyer** [comment]
No, only that you add yourself in the maintainers in a separate commit.

**Comment by BadDecisionsAlex** [comment]
Merging multiple PRs

**Pull request closed without merging**